docker-compose.yml

```diff
@@ -43,11 +43,7 @@ services:
         php-fpm
       "
     volumes:
-      - ./config:/application/config:rw
-      - ./src:/application/src
-      - ./public:/application/public
-      - ./templates:/application/templates
-      - ./tests:/application/tests
+      - .:/application
```

Comment:
this is the solution that I had proposed a few months ago that @fabiocicerchia rejected me. I err?